README.md

```diff
@@ -183,6 +183,16 @@ message Proto3Required {
 }
 ```
 
+### Custom schema file extention
+
+The default file extention is `jsonschema`. You can override that with the "file_extention" parameter.
```

Comment:
This is great to document. Thanks!

What would it take to say the default file extension for JSON schema to be `.json`? I think this would have many benefits with tools / IDEs / Chrome DevTools / GitHub that recognize the `.json` extension since the format is valid JSON.

IETF docs also use the `.json` extension when defining the spec, although I don't believe there is any requirement.

https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-00
https://json-schema.org/understanding-json-schema/basics.html

Reply:
OK I changed the default. I hope it doesn't piss too many people off, but at least they can override it with the optional parameter.

Reply:
Thanks for listening! Yeah, I realize it's a breaking change. I hope folks think it's for the better too!